livechat/webhooks/v33.py

```diff
@@ -0,0 +1,238 @@
+''' API v3.3 webhooks data classes. '''
+
+from dataclasses import dataclass
+
+# pylint: disable=missing-class-docstring
+
+
+@dataclass
+class WebhookV33:
+    webhook_id: str
+    secret_key: str
+    action: str
+    license_id: int
+    additional_data: dict
+    payload: dict
+
+    def payload_data_class(self):
```

Comment:
Missing returned type.

Reply:
I've deliberately didn't add returned type. The only way to actually add it would be to list all possible classes so sth like:
`-> Union[IncomingChat, ChatDeactivated, ChatAccessGranted, ChatAccessRevoked, [...]]`
It would be very long. I don't think it makes sense to do so. WDYT?